src/julia/devcontainer-feature.json

```diff
@@ -13,6 +13,15 @@
             ],
             "default": "release",
             "description": "Select a Juliaup channel"
+        },
+        "additionalChannels": {
+            "type": "string",
+            "default": "",
+            "description": "Additional Juliaup channels to install, separated by commas (e.g., 'nightly,1.10')",
+			"proposals": [
+                "",
+ 				"lts,nightly",
+ 			]
```

Comment:
It seems to be indented with tabs, not spaces. Could you change the indentation to spaces?
(I believe this should be checked by CI, but so far it has not been set up, sorry.)